perceval/simulators/simulator.py

```diff
@@ -392,6 +397,8 @@ def _probs_svd_fast(self, input_dist, p_threshold, progress_callback: Callable |
                 exec_request = progress_callback(progress, 'recombine distributions')
                 if exec_request is not None and 'cancel_requested' in exec_request and exec_request['cancel_requested']:
                     raise RuntimeError("Cancel requested")
+        if self._logical_perf > 0:
```

Comment:
Can you add the details of the formula to prove this is the good one in comments? I spent one hour to prove the formula is correct...

Reply:
Not sure I've added a mathematical grade proof, but I explained the computation steps in comments.